source/manuals/programming-languages/java.html.md.erb

```diff
@@ -137,7 +137,7 @@ We encourage the use of static analysis. Static analysis tools for Java include
 
 If possible, configure your static analysis tools using configuration files and keep these in your project repositories. This makes your settings more portable and makes it easier to perform the same checks in different places (for example, in a cloud service and on your own computer).
 
-Some cloud-based static analysis tools, including Codacy, can work directly with GitHub repositories. Only grant a tool access to your repositories if it has been approved by an information assurance review.
+Some cloud-based static analysis tools, including Codacy, can work directly with GitHub repositories. Only grant a tool access to your repositories if it has been approved by an information assurance review. Some teams are using [SonarCloud](https://sonarcloud.io/), which has been approved.
```

Comment:
It has been approved specifically for public repos only, which SonarCloud provides for free (at least, within Digital Identity)

Reply:
Rewritten that sentence to make the scope of SonarCloud approval clearer.

Reply:
Thanks!